runtime-transaction/benches/process_compute_budget_instructions.rs

```diff
@@ -0,0 +1,141 @@
+#![feature(test)]
+extern crate test;
+use {
+    solana_runtime_transaction::instructions_processor::process_compute_budget_instructions,
+    solana_sdk::{
+        compute_budget::ComputeBudgetInstruction,
+        instruction::Instruction,
+        message::Message,
+        pubkey::Pubkey,
+        signature::Keypair,
+        signer::Signer,
+        system_instruction::{self},
+        transaction::{SanitizedTransaction, Transaction},
+    },
+    test::Bencher,
+};
+
+const NUM_TRANSACTIONS_PER_ITER: usize = 1024;
+
+fn build_sanitized_transaction(
+    payer_keypair: &Keypair,
+    instructions: &[Instruction],
+) -> SanitizedTransaction {
+    SanitizedTransaction::from_transaction_for_tests(Transaction::new_unsigned(Message::new(
+        instructions,
+        Some(&payer_keypair.pubkey()),
+    )))
+}
+
+#[bench]
+fn bench_process_compute_budget_instructions_empty(bencher: &mut Bencher) {
+    let tx = build_sanitized_transaction(&Keypair::new(), &[]);
+    bencher.iter(|| {
+        (0..NUM_TRANSACTIONS_PER_ITER).for_each(|_| {
+            assert!(
+                process_compute_budget_instructions(tx.message().program_instructions_iter())
```

Comment:
we should black_box the tx so compiler doesn't optimize knowing this is same tx over and over.

Reply:
767c6f6e58a6942fd100e302404b92d1575d324b